count.json

```diff
@@ -1,4 +1,4 @@
 {
```

Comment:
Can you justify or explain this change?

Reply:
I'm not too sure why that's there or what the actions-user bot is. It seems there is another action that runs and makes that commit, but I didn't set that up, it was in the repo when I forked it.

Reply:
This is probably something similar to dependabot. I see the same on this fork : 
https://github.com/mayhemheroes/switchboard/commit/72e75a79c72f8cffdbcab3ca33f56102ee6a5700

I bet it changes depending on if it is a fork or upstream. In either case it's different from what is in this fork, maybe you could sync from this fork, or upstream?

Reply:
Okay, I changed that file back to match the fork you linked. I had to disable to workflow to keep it from updating the file continuously. Let me know if everything looks good now.